extensions/panache/hibernate-orm-panache/runtime/pom.xml

```diff
@@ -39,6 +39,36 @@
             <artifactId>quarkus-jackson</artifactId>
             <optional>true</optional>
         </dependency>
+
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-jpamodelgen</artifactId>
+            <scope>provided</scope>
+            <!-- Exclude these dependencies, since the are banned -->
```

Comment:
```suggestion
            <!-- Exclude these dependencies, since they are banned -->
```

Reply:
sure, I will do this change this evening.